README.md

```diff
@@ -26,104 +26,41 @@ as many questions have already been asked and answered. Please also
 the mailing list and post your new questions there.
 
 
-## How to Build GNU Radio
+## How to Install GNU Radio
+
+### Prebuilt Binaries
+
+The recommended way to install GNU Radio on most platforms is using available binary package distributions. 
+
+The following command is for Debian, Ubuntu, and derivatives. It will install Release 3.7 with Python2. 
```

Comment:
Since maint-3.8 isn't as much of a moving target, this needs a 

> (as of DEBIAN_VERSION, UBUNTU_VERSION)

We're obviously expecting things to start moving.

Also, since this is maint-3.8, this should explicitly link to Josh's PPA! That's the recommended way these days.

Reply:
If people only need the native packaged version, is that a true statement? In any case, I would prefer we refer to the wiki here, and only have build-from-source-related instructions checked into git.